Mathlib/Topology/Separation.lean

```diff
@@ -2100,6 +2166,36 @@ lemma SeparatedNhds.of_isCompact_isClosed {s t : Set X}
 @[deprecated (since := "2024-01-28")]
 alias separatedNhds_of_isCompact_isClosed := SeparatedNhds.of_isCompact_isClosed
 
+/-- This technique to produce a SeparatingCover in regular Lindelöf topological spaces
+will be used to prove regular Lindelöf spaces are normal. -/
+lemma SeparatingCover.of_isClosed {s t : Set X} [r: RegularSpace X] [LindelofSpace X]
```

Comment:
This name is the wrong way around, it should be `IsClosed.SeparatingCover` or, better, `IsClosed.HasSeparatingCover` so that you can use it with dot notation.

Reply:
5b96863